MockKit/MockCGMManager.swift

```diff
@@ -249,12 +341,18 @@ public final class MockCGMManager: TestingCGMManager {
             let currentValue = samples.first
         {
             switch currentValue.quantity.doubleValue(for: HKUnit.milligramsPerDeciliter) {
-            case ..<mockSensorState.lowGlucoseThreshold.doubleValue(for: HKUnit.milligramsPerDeciliter):
+            case ..<mockSensorState.cgmLowerLimit.doubleValue(for: HKUnit.milligramsPerDeciliter):
+                mockSensorState.glucoseValueType = .belowRange
+            case mockSensorState.cgmLowerLimit.doubleValue(for: HKUnit.milligramsPerDeciliter)..<mockSensorState.urgentLowGlucoseThreshold.doubleValue(for: HKUnit.milligramsPerDeciliter):
```

Comment:
[nit] I believe that the urgent low alert state typically triggers at less than *or equal to* the urgent low value (eg. 55). Consider ditching the ranges and just use comparison with <=.